inference-engine/src/auto_plugin/auto_plugin.cpp

```diff
@@ -24,20 +26,66 @@ namespace {
         return config;
     }
 
-    DeviceInformation SelectDevice(const std::vector<DeviceInformation>& metaDevices) {
-        for (auto& item : metaDevices) {
-            if (item.deviceName.find("CPU") == 0) {
-              return item;
+    std::string GetNetworkPrecision(const InferenceEngine::CNNNetwork &network) {
+        auto nGraphFunc = network.getFunction();
+        bool isINTModel = ngraph::op::util::has_op_with_type<ngraph::op::FakeQuantize>(nGraphFunc);
+        if (isINTModel) {
+            return "INT8";
```

Comment:
can we use `METRIC_VALUE(INT8)` instead of hardcoded string? the same for FP16 and FP32

Reply:
done